src/transform/WidenImage.js

```diff
@@ -28,6 +28,9 @@ const ancestorsToWiden = element => {
  * @return {void}
  */
 const updateStyleValue = (style, key, value) => {
+  if (!style || !key) {
```

Comment:
The JSDocs indicate these states are impossible when called correctly. Maybe this is a symptom of us using `el.parentNode` instead of `el.parentElement` since only Elements have a style property?